Devscord.DiscordFramework/Services/MessagesService.cs

```diff
@@ -78,17 +73,16 @@ public async Task SendFile(string fileName, Stream stream)
             await stream.DisposeAsync();
         }
 
-        public async Task SendExceptionResponse(BotException botException)
+        public Task SendExceptionResponse(BotException botException)
         {
             var responseName = botException.GetType().Name.Replace("Exception", "");
             var responseManagerMethod = typeof(ResponsesManager).GetMethod(responseName);
             if (responseManagerMethod == null)
             {
                 Log.Error("{name} doesn't exists as a response", responseName);
-                await this.SendMessage($"{responseName} doesn't exists as a response"); // message typed into code, bcs it's called only when there is a problem with responses
-                return;
+                return this.SendMessage($"{responseName} doesn't exists as a response"); // message typed into code, bcs it's called only when there is a problem with responses
```

Comment:
wrzuć do responses

Reply:
no ale nie lepiej aby to jednak pozostało w kodzie?
bo w końcu ten komunikat ma być pokazany tylko w momencie kiedy jest problem z responses albo czegoś nie ma
więc jak ma wyświetlić komunikat o problemie kiedy nie będzie mógł tego pobrać z responses?

Reply:
dobrze będzie dodać testy sprawdzające takie rzeczy (analiza refleksjami czy kod ma prawo zadziałać)

ale to w przyszłości